tests/Microsoft.Graph.DotnetCore.Core.Test/Exceptions/ErrorTests.cs

```diff
@@ -126,11 +126,13 @@ public void Validate_ErrorObjectDeserializes()
             Assert.Equal("inner-error-code", error.InnerError.Code);
             Assert.Equal("target-value", error.Target);
             Assert.NotNull(error.AdditionalData);
-            Assert.Equal("unexpected-property-value", error.AdditionalData["unexpected-property"]);
+            var expectedValue1 = error.AdditionalData["unexpected-property"].ToString();
```

Comment:
>expectedValue1 [](start = 16, length = 14)

why introduce the variable instead of just adding the ToString() in the Assert.Equal

Reply:
Resolved